R/lru.R

```diff
@@ -28,7 +35,16 @@ LRUcache_class <- R6::R6Class("LRUcache",
   private = list(
     max_num = 0,
     data    = new.env(),
+    use_bytes = FALSE,
     save    = function(name, value) {
+      # Check if this value alone exceeds the cache size
+      size <- if(private$use_bytes) as.integer(as.character(pryr::object_size(value))) else 1
+      if (size > private$max_num){
+        warning(sprintf("'%s' is too large (%dB) to fit in the cache (%dB) and will not be cached.  Consider creating a larger cache.", name, size, private$max_num), call. = FALSE)
```

Comment:
tiny comment - you have double space before 'Consider'. Great use of sprintf though :+1:, I'd have slacked on that :)